### PR TITLE
Include node() in getting TS table type status

### PR DESCRIPTION
**Pull request by paegun**
* change to include the node in getting the (TS table) type status.

### CODE REVIEW

File: src/riak_core_claimant.erl [hunk 1]
```diff
@@ -719,7 +719,7 @@ all_members() ->
 -spec get_remote_type_status(BucketType::binary()) ->
     {AllProps::[any()], BadNodes::[node()]}.
 get_remote_type_status(BucketType) ->
-    rpc:multicall(all_members() -- [node()],
+    rpc:multicall(all_members(),
```
Comment (macintux) [inline src/riak_core_claimant.erl:722]: Given that we only call this from one spot in the claimant, I'd like to see the function renamed.

Comment (macintux) [reply src/riak_core_claimant.erl:722]: I guess that means we should rename `get_remote_ddl_compiled_status` for consistency.

**Comment by macintux** [comment]
+1

**Comment by hmmr** [comment]
+1

**Comment by macintux** [comment]
No Thumbs, merging.

**Pull request merged**